src/metabase/models/recent_views.clj

```diff
@@ -25,7 +25,7 @@
 (def ^:private ^:dynamic *recent-views-stored-per-user*
   "The number of recently viewed items to keep per user. This should be larger than the number of items returned by the
   /api/activity/recent_views endpoint, but it should still be lightweight to read all of a user's recent views at once."
-  100)
+  30)
```

Comment:
Dropping this value to a more reasonable one, just in case this was the cause of any perf issues.

Reply:
Is there any evidence that this is related to a perf issue? I am chesterton's-fencing over it, since I don't know why it was 100 in the first place.